test/cmake-test.el

```diff
@@ -0,0 +1,68 @@
+;;; cmake-tests.el --- company-mode tests  -*- lexical-binding: t -*-
+
+;; Copyright (C) 2016  Free Software Foundation, Inc.
+
+;; Author: oracleyue
```

Comment:
We'd prefer your full name here, I think. But also, this amount of code (and even without the tests) will need copyright assignment. Are you willing to sign the papers?

Reply:
Okay, I will update the name now.
To be honest, I don't mind the copyright issue (just a small piece of codes to fix bugs). If you prefer everything being strictly legal, please let me know and I will follow your instructions.

Reply:
You might also directly include the codes. No problem for me. Feel free to choose any way you prefer to deal with it.

Reply:
It's Emacs project policy, that's all. Please see the instructions here: http://orgmode.org/cgit.cgi/org-mode.git/plain/request-assign-future.txt

Reply:
Okay, I will do it. Thx!

Reply:
I have sent the email. No idea what's gonna receive. I am waiting for their response. 

It's my time first to get involved in a collaborated project (I only dealt with my own codes with a few lines declaring GPL or offered codes directly without pull requests). Thanks a lot for your patience on telling me so much to process them!